src/pages/fr/guides/styling.md

```diff
@@ -0,0 +1,296 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: CSS et Stylisation
+description: Apprenez à utiliser les composants de style avec Astro.
+setup: |
+  import Since from '../../../components/Since.astro';
+---
+
+Astro à été conçu pour rendre la création de style et l'écriture de CSS facile. Écrivez votre propre CSS directement dans un composant Astro ou importez une bibliothèque de style préférée comme [Tailwind][tailwind]. Les langages de style avancés comme [Sass][sass] et [Less][less] sont également supportés.
+
+## Stylisation dans Astro
+
+Styliser un composant Astro est aussi facile que d'ajouter une balise `<style>` à votre template de composant ou de page. Quand vous placez une balise `<style>` dans un composant Astro, Astro détectera le CSS et chargera vos styles pour vous, automatiquement.
+
+```astro
+<style>
+  h1 { color: red; }
+</style>
+```
+
+### Portée des styles
+
+Les règles CSS `<style>` d'Astro sont automatiquement **portées au composant par défaut**. Ces styles sont compilés en arrière-plan pour ne s'appliquer qu'à l'HTML écrit à l'intérieur du même composant. Le CSS que vous écrivez dans un composant Astro est automatiquement encapsulé dans ce composant.
+
+```diff
+<style>
+-  h1 { color: red; }
++  h1.astro-HHNQFKH6 { color: red; }
+-  .text { color: blue; }
++  .text.astro-HHNQFKH6 { color: blue; }
+</style>
+```
+
+Les styles portés ne se propagent pas et ne n'affectent pas le reste de votre site. Dans Astro, c'est quelque chose de courrant d'utiliser des sélecteurs de basse-précision comme `h1 {}` ou `p {}` car ils seront compilés avec des encapsulations dans la sortie finale.
+
+Les styles portés ne s'appliquent pas à d'autres composants Astro contenus dans votre template. Si vous avez besoin de styliser un composant enfant, envisagez de placer ce composant dans une `<div>` (ou d'autres éléments) que vous pourrez ensuite customiser.
+
+#### Styles globaux
+
+Comme nous recommandons les styles portés pour la plupart des composants, vous pourriez éventuellement trouver une raison valide de écrire du CSS global, non porté. Vous pouvez désactiver l'encapsulation automatique de CSS avec l'attribut `<style is:global>`.
```

Comment:
Même si nous recommandons les styles délimités pour la plupart des composants, vous pouvez éventuellement trouver une raison valide d'écrire du CSS non délimité global. Vous pouvez désactiver l'encapsulation automatique de CSS avec l'attribut `<style is:global>`.